docs/docs_skeleton/docs/modules/chains/document/refine.mdx

```diff
@@ -5,8 +5,8 @@ sidebar_position: 1
 
 The refine documents chain constructs a response by looping over the input documents and iteratively updating its answer. For each document, it passes all non-document inputs, the current document, and the latest intermediate answer to an LLM chain to get a new answer.
 
-Since the Refine chain only passes a single document to the LLM at a time, it is well-suited for tasks that require analyzing more documents than can fit in the model's context.
-The obvious tradeoff is that this chain will make far more LLM calls than, for example, the Stuff documents chain.
-There are also certain tasks which are difficult to accomplish iteratively. For example, the Refine chain can perform poorly when documents frequently cross-reference one another or when a task requires detailed information from many documents.
+Since the refine chain only passes a single document to the LLM at a time, it is well-suited for tasks that require analyzing more documents than can fit in the model's context.
```

Comment:
Refine, Stuff - those are terms, names of the chains. They should be capitalized.
Also L9, L10